web/react-web/src/Graph/Graph.js

```diff
@@ -0,0 +1,241 @@
+import React, {Component} from 'react';
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+import {AXES_LINEAR, AXES_LOG, AXES_LOG_LOG} from "../sharedObjects";
+
+const colors = ["#7cb5ec", "#434348", "#90ed7d", "#f7a35c", "#8085e9", "#f15c80", "#e4d354", "#2b908f", "#f45b5b", "#91e8e1"];
+
+const predefinedOptions = {
+    chart: {
+        zoomType: 'x',
+        colors,
+    },
+    title: {
+        text: ''
```

Comment:
We can set `COVID 19 Chart` or something like that, no? (No idea where it shows up, but somewhere it does I guess) :)

Reply:
It's on top of chart - main title, but it's changed every time you pick some country. This line is just predefined text. The whole concept is, that I have initial config for highcharts `const predefinedOptions = {...` and during render I change only what's needed. Title is changing.